test/windows/rpm.test.ts

```diff
@@ -0,0 +1,51 @@
+import * as path from "path";
```

Comment:
Maybe we don't need the rpm tests specifically, I think we already test those above for the OS + 2 apps test case (I see it already uses amazonlinux:2)